satpy/etc/readers/slstr_l2.yaml

```
@@ -0,0 +1,59 @@
+reader:
+  description: NC Reader for Sentinel-3 SLSTR data
```

```suggestion
  description: NC Reader for Sentinel-3 SLSTR Level 2 data
```
Or even: is it just Sea Ice and SST in these files, or is there more ?

---

There are other data that I believe are ECMWF auxiliary data.